packages/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -11,15 +11,22 @@ export enum TextAreResizeOrientation {
   both = 'both'
 }
 
+export enum TextAreaReadOnlyVariant {
+  default = 'default',
+  vertical = 'plain'
+}
+
```

Comment:
Is this needed? I don't see it referred to anywhere, and I was curious what "vertical" means.

Reply:
@thatblindgeye suggested we export an enum here:

> [...] we could create a ReadOnlyVariant enum to export, similar to alert's AlertVariant [...]
_Originally posted by @thatblindgeye in https://github.com/patternfly/patternfly-react/pull/7996#pullrequestreview-1113924883_

Updated the enum here with the correct keys (i.e. `default` and `plain`).